docs/docs/articles/github-actions.md

```diff
@@ -1,135 +1,85 @@
-# GitHub Actions
-
-In order to automate Testkube runs, access to a K8s cluster is needed. For example, a configured environment with the set up context and kubeconfig for communication with the K8s cluster.
-
-Testkube uses your K8s context and access settings in order to interact with the cluster and tests resources, etc.
-
-In the next few sections, we will go through the process of Testkube and Helm (for Testkube's release deploy/upgrade) automations with the usage of GitHub Actions and GKE K8s.
-
 ## Testkube GitHub Action
 
-The testkube GitHub Action is available here <https://github.com/kubeshop/testkube-docker-action> and it enables running the Testkube CLI commands in a GitHub workflow.
+The Testkube GitHub Action installs the Testkube client and enables running any [Testkube CLI](https://docs.testkube.io/cli/testkube) command in a GitHub workflow. It is available on Github Marketplace <https://github.com/marketplace/actions/testkube-action>.
```

Comment:
> GitHub Action installs the Testkube client

What do we mean by Client on the first line? not sure users will be confused about this one. The Testkube action requires the agent to be already deployed so I'm not sure what thing is the github action installing.

Reply:
I mean CLI, will remove `client` not to confuse users

Reply:
not sure if installing the CLI the correct term 🤔
is it installed every time the pipeline runs and where ?

Reply:
It's a correct term - it's installed every time on the runner.